pkg/controller/operators/olm/operator.go

```diff
@@ -102,12 +103,57 @@ type Operator struct {
 	clientFactory                clients.Factory
 	plugins                      []plugins.OperatorPlugin
 	informersByNamespace         map[string]*plugins.Informers
+	informersFiltered            bool
+
+	ruleChecker     func(*v1alpha1.ClusterServiceVersion) *install.CSVRuleChecker
+	ruleCheckerLock *sync.RWMutex
```

Comment:
I think this can be a non-pointer unless you're passing it into other places or you have non-pointer receivers in here?

Reply:
Good point - I think I was passing it in a previous impl.